File: lab spring25/assignment1/taskE.cpp
#include <iostream>
#include <vector>
using namespace std;
#define br cout << '\n'

void bubbleSort(vector<long long> &arr){
	bool sorted = true;
	for(int i=0;i<arr.size()-1;i++){
    	sorted = true;
    	for(int j=0;j<arr.size()-1-i;j++){
        	if(arr[j]>arr[j+1]) {
            	swap(arr[j],arr[j+1]);
            	sorted = false;
        	}
    	}
    	if(sorted) return;
	}
}                                              	 
    
void solve(){
	int n;
	cin >> n;
	vector<long long> arr(n);
	for(int i=0;i<n;i++) cin >> arr[i];
	bubbleSort(arr);
	for(int i=0;i<n;i++) cout << arr[i] << ' ' ;br;
}
  
int main(){
	freopen("input.txt","r",stdin);
	solve();
}

File: lab summer25/assignment4/taskE.py
import sys
sys.stdin = open("input.txt","r")

def solve():
    n,m = list(map(int,input().split()))
    u = list(map(int,input().split()))
    v = list(map(int,input().split()))
    ans = [0]*(n+1)
    for i in range(m):
        ans[v[i]]+=1
        ans[u[i]]-=1
    for i in range(1,n+1): print(ans[i],end=' ')
    print()


t = 1
t = int(input())
while t>0:
    t-=1
    solve()